lang/cs/Org.Apache.REEF.Tests/Functional/Common/Task/ExceptionTask.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+// 
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+using System;
+using Org.Apache.REEF.Common.Tasks;
+
+namespace Org.Apache.REEF.Tests.Functional.Common.Task
+{
+    public abstract class ExceptionTask : ITask
+    {
+        private readonly Exception _exToThrow;
+
+        protected ExceptionTask(Exception exToThrow)
+        {
+            _exToThrow = exToThrow;
+        }
+
+        public byte[] Call(byte[] memento)
+        {
+            throw _exToThrow;
+        }
+
+        public void Dispose()
+        {
+            throw new System.NotImplementedException();
```

Comment:
Right, should not throw exception. Dispose will be called from some finally block to release resources. If this task has nothing to release, just leave it empty.